### ISSUE TITLE
Search an implementation method's reference of an interface doesn't have the result of the reference from the other assembly.

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#28510]
I have a solution, and there are 2 projects in the solution: 

Project A, .net standard 2.0 library project, contains an interface IC. And there is a usage of interface IC.

Project B, WPF application, .net framework 4.7.1, has dependency of project A, and there is a class, implemented from IC.

In the interface IC, there is a method MethodD.

When I try to find all the references of MethodD, it didn't return the reference in Project A. Only the result of reference in Project B is found.

_This issue has been moved from https://developercommunity.visualstudio.com/content/problem/291520/search-an-implementation-methods-reference-of-an-i.html
VSTS ticketId: 646133_
_These are the original issue comments:_
(no comments)
_These are the original issue solutions:_
(no solutions)

**Comment by sharwell** [comment]
IIRC, I used to observe problems with this when working on the `ITemporaryStorageService` interface in Roslyn. The problems appeared on source.roslyn.io, making it impossible to find the implementations of that interface which we actually used, instead finding only the ones in test code and the trivial fallback implementation. However, the issues on source.roslyn.io appear to have since been resolved.
